webapp/src/components/Staking/Modal/StakingActionModal.tsx

```diff
@@ -462,20 +464,23 @@ const StakingActionModal: React.FC<StakingActionModalProps> = ({
               </BaseModalContentColumn>
             ) : (
               <BaseModalContentColumn marginTop="auto">
-                <BaseUnderlineLink
-                  to={`${getEtherscanURI()}/tx/${txId}`}
-                  target="_blank"
-                  rel="noreferrer noopener"
-                  className="d-flex"
-                >
-                  <PrimaryText className="mb-2">View on Etherscan</PrimaryText>
-                </BaseUnderlineLink>
+                {chainId && (
+                  <BaseUnderlineLink
+                    to={`${getEtherscanURI(chainId)}/tx/${txId}`}
+                    target="_blank"
+                    rel="noreferrer noopener"
+                    className="d-flex"
+                  >
+                    <PrimaryText className="mb-2">View on Etherscan</PrimaryText>
```

Comment:
SnowTrace?

Reply:
There's no staking on avax, but this can be added.